Changelog.md

```diff
@@ -1,5 +1,54 @@
 # Changelog
 
+## Version 1.2.0
+* Revise fsdev initialization
+  * Removed fsdevInit/Exit
+  * Now automatically initializing fsdev state on first mount
+  * Added fsdevMountSdmc (replaces fsdevInit)
+  * Added fsdevUnmountAll (replaces fsdevExit)
+* Use Service for all fs sessions. This fixes an issue with savedata commit.
+* Implemented FsSaveDataIterator (aka ISaveDataInfoReader).
+* improve usbComms support
+  * Allow using multiple interfaces.
+  * Added usbComms Ex funcs.
+  * Use RwLock.
+* hid updates
+  * Added hidGetHandheldMode().
+  * Added hidSetNpadJoyAssignmentModeSingleByDefault()
+  * Added hidSetNpadJoyAssignmentModeDual().
```

Comment:
Should insert the text from commit-msg e7a4eba71c2aada52182a977c2068667b16b9f2a "Use...", after this line.